pxl_scripts/update_readme.py

```diff
@@ -1,12 +1,16 @@
+# Run using python3.
+
 import os
 import yaml
 
+if os.getcwd().split("/").pop() == "pixie":
+	os.chdir("./pxl_scripts")
 
 def script_list_format(folder, script, desc):
 	github_url = "https://github.com/pixie-labs/pixie/tree/main/pxl_scripts"
 	return f"- {folder}/[{script}]({github_url}/{folder}/{script}): {desc}"
 
-def get_script_list(directory="px"):
+def get_script_list(directory="./px"):
```

Comment:
similarly, the directory to scan can probably be a path as well. 

Alternatively, you can use a glob to find all the **/manifest.yaml files and then use that to determine what the active scripts are.